### DISCUSSION TITLE
Auto-completion or a form for GRPC args

### DISCUSSION THREAD

**Discussion by 06kellyjac** [Kong/insomnia#6358] [Ideas]
With a GRPC proto definition insomnia knows exactly what elements are required for the message

```
syntax = "proto3";

package search.v1;

message SearchRequest {
  string query = 1;
  int32 page_number = 2;
  int32 results_per_page = 3;
}

service SearchService {
  rpc Search(SearchRequest) returns (SearchResponse);
}
```

If I feed this to insomnia it can give me a dropdown with `/SearchService/Search` but then it just gives me `{}` to fill in myself.
Insomnia knows that this request wants `query` of `string`, and `page_number` or `results_per_page` as `int32` but I don't get any autocompletion or any warnings when I add a superfluous `hello` field.

I'd like it to either help with writing out the body, or have a form style input where I can fill out the parameters

I expected this to already exist so I've had a look around in the docs, issues, and discussions but this doesn't seem to currently be a feature or anything raised before.

**Comment by ThaDaVos** [comment]
Really stupid this isn't already done - like you said, the foundation is there, it knows this information already...

**Reply by ThaDaVos** [reply]
@06kellyjac  - just to note - there is a button which allows you to fill it with an example - but the autocomplete is still missing
